src/Authorization/Traits/Authorizable.php

```diff
@@ -405,4 +430,13 @@ private function getConfigPermissions(): array
             ? array_keys(setting('AuthGroups.permissions'))
             : array_keys(config('AuthGroups')->permissions);
     }
+
+    /**
+     * modifies the banned status of the user
+     */
+    private function modifyBanStatus(): void
+    {
+        $model = model(UserModel::class);
```

Comment:
I believe you can get the User Provider (which might be a class that extends the UserModel) directly from the `auth()` helper. Double-check me on that, though. 

```suggestion
        $model = auth()->getProvider();
```